prow_config.yaml

```diff
@@ -23,3 +92,62 @@ workflows:
       - components/crud-web-apps/common/frontend/kubeflow-common-lib/*
       - components/crud-web-apps/jupyter/*
     kwargs: {}
+  # Create and push Access Management OCI image to ECR
+  - py_func: kubeflow.kubeflow.cd.access_management.create_workflow
+    name: a-man-build
+    job_types:
+      - postsubmit
+    include_dirs:
+      - components/access-management/*
+    kwargs: {}
+  # Create and push Admission Webhook OCI image to ECR
+  - py_func: kubeflow.kubeflow.cd.admission_webhook.create_workflow
+    name: ad-wh-build
+    job_types:
+      - postsubmit
+    include_dirs:
+      - components/admission-webhook/*
+    kwargs: {}
+  # Create and push Central Dashboard OCI image to ECR
+  - py_func: kubeflow.kubeflow.cd.central_dashboard.create_workflow
+    name: cdash-build
+    job_types:
+      - postsubmit
+    include_dirs:
+      - components/centraldashboard/*
+    kwargs: {}
+  # Create and push Tensorboard Web App OCI image to ECR
+  - py_func: kubeflow.kubeflow.cd.twa.create_workflow
+    name: twa-build
+    job_types:
+      - postsubmit
+    include_dirs:
+      - components/crud-web-apps/common/frontend/kubeflow-common-lib/*
```

Comment:
lets also include the directory of the common code here as well
`components/crud-web-apps/common/backend`